ldap_auth_provider.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, config, account_handler):
         self.ldap_active_directory = config.active_directory
         if self.ldap_active_directory:
             self.ldap_default_domain = config.default_domain
+            self.ldap_root_domain = False
```

Comment:
It is worth commenting that this can be `False`, `None`, or a string, which is obviously without reading much of the code.